webview-ui/src/i18n/locales/en/kilocode.json

```diff
@@ -208,24 +208,22 @@
 	"ghost": {
 		"title": "Autocomplete",
 		"settings": {
-			"autoTrigger": "Auto Trigger",
 			"triggers": "Triggers",
-			"keybindingDescription": "Autocomplete features can be triggered by commands. You can configure the <DocsLink>Keyboard Shortcuts here</DocsLink>.",
 			"enableAutoTrigger": {
-				"label": "Enable Autocomplete",
-				"description": "When enabled, Kilo Code will automatically trigger Autocomplete when you stop typing. This can be useful for quick fixes and suggestions."
+				"label": "Pause to Complete",
+				"description": "When enabled, Kilo Code will automatically trigger Autocomplete when you pause typing. This can be useful for quick fixes and suggestions."
 			},
 			"autoTriggerDelay": {
 				"label": "Auto Trigger Delay",
-				"description": "The delay in seconds before Kilo Code triggers Autocomplete after you stop typing. A shorter delay means quicker suggestions, but may be more resource-intensive."
+				"description": "The delay in seconds before Kilo Code triggers Autocomplete after you pause typing. A shorter delay means quicker suggestions, but may be more resource-intensive."
 			},
 			"enableQuickInlineTaskKeybinding": {
-				"label": "Quick Task",
-				"description": "Only need a quick change? Select some code (or don't!) and hit Cmd+I. Describe your goal in plain English (\"create a React component with these props\", \"add error handling to this function\"), and get multiple, ready-to-use suggestions right in your editor. <DocsLink>See shortcut</DocsLink>"
+				"label": "Quick Task (keyboard shortcut)",
```

Comment:
@hassoncs my idea is that where it says keyboard shortcut we actually fill in the shortcut as retrieved by your code

Reply:
(but I'd say it's also fine to merge as is for now)